test/conformance/BUILD

```diff
@@ -31,22 +38,35 @@ filegroup(
 )
 
 genrule(
-    name = "list_conformance_tests",
+    name = "list_conformance_specs",
     srcs = [
         "//test/e2e:all-srcs",
         "//test/e2e_node:all-srcs",
+        "//test/e2e:e2e.test_binary",
+        "//vendor/github.com/onsi/ginkgo/ginkgo",
     ],
-    outs = ["conformance.txt"],
-    cmd = "./$(location :conformance) $(locations //test/e2e:all-srcs) > $@",
+    outs = ["specsummaries.json"],
+    cmd = "$(location //vendor/github.com/onsi/ginkgo/ginkgo) --dryRun=true --focus=[Conformance] $(location //test/e2e:e2e.test_binary) -- --spec-dump $$(pwd)/$@ > /dev/null",
```

Comment:
I think we could simulate excluding invalid tags by adding a skip regex here

Reply:
If you did that, it would not dump their test info leading to them not resulting in the final doc. However, it wouldn't stop the test from actually being executed with `ConformanceIt(...` which would mean it could lead to the tests chosen by `--focus=[Conformance]` not lining up with the tests described in the doc generated by this.